.github/workflows/stale.yml

```diff
@@ -0,0 +1,29 @@
+name: Close Stale Pull Requests
+
+on:
+  schedule:
+    # Runs at 00:00 every day
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-18.04
+
+    steps:
+     - uses: actions/checkout@v2.0.0
+     - name: stale
+       uses: gatsbyjs/stale@v1.1.0
+       with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Will this work for forked PRs as well?

Reply:
> If it's not a limitation of GitHub's API, then we can fork and upgrade the project.

Don't think it a GitHub API limitation, since other actions provide the same feature.

I will look into it in another PR? Cause will take a bit of time to figure that out.

Reply:
Do u mean this [repo](https://github.com/Tejas2805/Github-Actions)? It seems like the labelling is working, but the closing is not tho.